modules/core/react/spec/InputProvider.spec.tsx

```diff
@@ -208,7 +208,8 @@ describe('InputProvider', () => {
         </InputProvider>
       </InputProvider>
     );
-    expect(component.find('.wdc-input-provider').length).toBe(1);
+
+    expect(document.querySelectorAll('[data-whatinput]').length).toBe(1);
```

Comment:
I feel like we need a better way of testing that these are behaving as a singleton. Now that they both point to `document.body`, even if they are both setting the attribute, there will only be one match to this query.